parsers/rhea/rhea.go

```diff
@@ -0,0 +1,791 @@
+package rhea
+
+import (
+	"compress/gzip"
+	"database/sql"
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
+
+/******************************************************************************
+
+Lower level structs
+
+These structs operate at the lowest level to parse the Rhea Rdf database dump
+into structs that Golang can understand. Getting all of Rhea from Rdf->Golang
+is quite verbose, and so most of the time you should not use these structs unless
+you know what you are doing and it cannot be accomplished with higher level structs.
+
+******************************************************************************/
+
+// Rdf is the RDF XML representation of the Rhea database.
+type Rdf struct {
+	XMLName      xml.Name      `xml:"RDF"`
+	Descriptions []Description `xml:"Description"`
+}
+
+// Description is the XML representation of an object in the Rhea database.
+type Description struct {
+	// Reaction
+	XMLName              xml.Name             `xml:"Description"`
+	About                string               `xml:"about,attr"`
+	ID                   int                  `xml:"id"`
+	Accession            string               `xml:"accession"`
```

Comment:
```suggestion
	Accession            string               `xml:"accession"` // Accession is the stable ID in the UniProt database. https://www.uniprot.org/help/accession_numbers
```

Could be helpful

Reply:
I will comment that, but with something different. Those accession numbers are not Uniprot accession numbers, they're Rhea accession numbers. More confusingly, you can "access" different things with each different accession number (some for reactions, some for compounds, etc etc etc). Only reaction accession numbers can be associated with Uniprot numbers using the TSV links that Rhea provides.